phpBB/docs/events.md

```diff
@@ -986,6 +986,34 @@ mcp_forum_actions_before
 * Since: 3.1.11-RC1
 * Purpose: Add some information before actions fieldset
 
+mcp_forum_last_post_author_username_append
+===
+* Locations:
+    + styles/prosilver/template/mcp_forum.html (2)
```

Comment:
What does `(2)` mean here and below?

Reply:
> What does `(2)` mean here and below?

Occurrence of the event in that file.

Reply:
Doubt it's allowed, but it's up to devs.

Reply:
> Doubt it's allowed, but it's up to devs.

Last time I was told to do that.

Reply:
@rxu ,

Just found,
check this : https://github.com/phpbb/phpbb/pull/5571

Best regards 👍